src/examples/typography/Ellipsis.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Ellipsis } from '@zendeskgarden/react-typography';
+
+const Example = () => (
+  <>
```

Comment:
Recommend replacing with `<MD>` in order to express Garden's intended `line-height`.